packages/server/src/api/controllers/static/templates/BudibaseApp.svelte

```diff
@@ -132,6 +132,7 @@
       <p />
     {/if}
   </div>
+  <script src="https://www.google.com/recaptcha/api.js"></script>
```

Comment:
Should we only load this if they have the feature enabled? I can imagine some people being a little upset about this being unconditionally loaded.